.github/FOUNDING.yml

```diff
@@ -0,0 +1,2 @@
+
+open_collective: openmage
```

Comment:
Should the file name be `FUNDING.yml`?

Reply:
it was this name when I added it for the composer installer

Reply:
argh yes, now I see it too 😨

Reply:
filename is corrected 🙈